lib/glossarist/concept.rb

```diff
@@ -42,7 +76,7 @@ def to_h
       {
         "termid" => id,
         "term" => default_designation,
-        "related" => related_concepts,
+        "related" => related&.map(&:to_h),
```

Comment:
There will be a problem when related concept is related to itself, i.e. circular relation. In the related concepts, we should likely only provide their unique identifiers.

In addition, we will need to serialise a Glossarist Term into JSON-LD (in the near future), so imagine using GraphQL to retrieve related concepts.

Reply:
`related` is an array containing the `RelatedConcept` objects. It only contains a reference to the related object. the schema for RelatedConcept is

```JSON
{
    "type": "<type of relation>",
    "content": "<String>",
    "ref": "<Ref>",
}
```

And Ref contains
```JSON
{
    "source": "<String>",
    "id": "<String>",
    "version": "<String>",
}
```

According to this schema, the reference will only contain the identifier.

Reply:
OK thanks!